### PR TITLE
Oop school library

### PR DESCRIPTION
**Pull request by Josphat205**
## Procedure

- Created a class Person
- Created class Student
- Created class Teacher

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+    - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub Actions:
+
+![gh actions checks](../assets/images/gh-actions-rubocop-linters-checks.png)
+
+Click on the `Details` link to see the full output and the errors that need to be fixed:
+
+![gh actions failing checks](../assets/images/gh-actions-rubocop-failing-checks.png)
```
Comment (brytebee) [inline README.md]: - [x] **Required:** These images are broken, kindly fix them to make your `README` look professional.

image

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  public
+
+  def can_use_services?
+    if @age >= 18 || @parent_permission
+      true
+    else
+      false
+    end
+  end
```
Comment (brytebee) [inline person.rb]: - [x] **Optional:** 
- Great work using the `public` access modifier, however, it is conventional to write all public methods before using the `private` access modifier.
- Moreso, the `@age >= 18 || @parent_permission`  would return a boolean, the `if-else` block is redundant.

File: student.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Student < Person
+  def initilize(classroom)
+    super()
```
Comment (brytebee) [inline student.rb]: - [x] **Required:** To get the required input from the parent class, restructure your `initialize()` as below:

```
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

File: teacher.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher < Person
+  def initilize(specialization)
+    super()
```
Comment (brytebee) [inline teacher.rb]: - Update as above.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Josphat205 & @lomantech👋,

Good job so far coming so far with the progress you've made! 💪💪

##

### Highlights💡
- Good PR Summary.✔️
- The `README` file is documented professionally.✔️`
- Good commit messages.✔️

 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Josphat205) [reply README.md]: Added proffesional readme

**Review by FrederickMih — APPROVED** [review]
Hi @Josphat205 , @brytebee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct GitHub workflow :heavy_check_mark: 
- No Linter error :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**